helpers/postgresql_dump.sh

```diff
@@ -29,6 +27,8 @@ katello-service start
 
 set +x
 
-echo "Now copy the generated files to the destination host:"
-echo "config_files.tar.gz, mongo_data.tar.gz, pulp_data.tar, "\
+echo "Now, copy the generated files to the destination host:"
+echo "config_files.tar.gz, mongo_data.tar.gz, pulp_data.tar,"\
      "candlepin.dump, foreman.dump"
+echo "Note: Do not copy the generated pg_sql_data.tar.gz as this is not"\
+     "required for migration process."
```

Comment:
@sthirugn Please specify `RHEL 6 to 7 migration` here as `migration` is a bit ambiguous